index.js

```diff
@@ -19,9 +22,19 @@ dotenv.config(); // Set up environment variables
 const app = express();
 const port = process.env.PORT || 3000;
 
+/* eslint-disable no-new */
+new Airlock({
+  server: app,
+  airtableApiKey: [apiKey],
+  airtableBaseId: BASE_ID,
```

Comment:
I would recommend getting the API key and BASE ID directly from `process.env`, seems a little but circuitous to import it from airtable? but I understand the intention so I don't mind either way